File: ios/My Job Pitch/My Job Pitch/Location.m
//
//  Location.m
//  My Job Pitch
//
//  Created by user on 12/01/2016.
//  Copyright © 2016 SC Labs Ltd. All rights reserved.
//

#import "Location.h"

@implementation Location

- (Image*)getImage
{
    if (self.images && self.images.count > 0) {
        return [self.images firstObject];
    } else {
        return [self.businessData getImage];
    }
}

@end

File: ios/My Job Pitch-Swift/MyJobPitch/Models/Location.swift
//
//  Location.swift
//  MyJobPitch
//
//  Created by dev on 12/20/16.
//  Copyright © 2016 myjobpitch. All rights reserved.
//

import Foundation

class Location: MJPObjectWithDates {

    var business: NSNumber!
    var jobs: NSArray!
    var name: String!
    var desc: String!
    var email: String!
    var telephone: String!
    var mobile: String!
    var emailPublic = false
    var telephonePublic = false
    var mobilePublic = false
    var placeName: String!
    var placeID: String!
    var longitude: NSNumber!
    var latitude: NSNumber!
    var address: String!
    var images: NSArray!
    var businessData: Business!
    
    func getImage() ->Image? {
        if images != nil && images.count > 0 {
            return images.firstObject as? Image
        }
        return businessData.getImage()
    }
}
